recipes/velocypack/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -u
+ferr(){
+    echo "$@"
+    exit 1
+}
+
+CXX_STANDARD=${CXX_STANDARD:-14}
+echo "requesting c++ standard ${CXX_STANDARD}"
+BUILD_TYPE=${BUILD_TYPE:-Release}
+
+mkdir build
+cd build
+python_path=$(which python)
```

Comment:
we already have an environment variable for the python interpreter at building time, you can use ``$PYTHON``

Reply:
thanks for the hint, I removed python_path=$(which python)